README.md

```diff
@@ -1,13 +1,7 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
-## The problem
-
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+A basic Netflix API built with Heroku and Express. 
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://molbimien-project-express-api.herokuapp.com 
```

Comment:
Tried it in both postman and this version and they both worked. I got a warning in my console though about the file being oversized "maxJsonSize" so I guess pagination might be good for this response. Or my computer is tired.